docs/standalone/batches.md

```diff
@@ -1,3 +1,55 @@
 ### Batches
```

Comment:
Just noticed, shouldn't this be under the Java Client section instead of Standalone?